src/libs/filtering.c

```diff
@@ -1664,6 +1670,11 @@ static gboolean _sort_init(_widgets_sort_t *sort, const dt_collection_sort_t sor
     gtk_box_pack_start(GTK_BOX(sort->box), sort->direction, FALSE, TRUE, 0);
     g_signal_connect(G_OBJECT(sort->direction), "toggled", G_CALLBACK(_sort_reverse_changed), sort);
     dt_gui_add_class(sort->direction, "dt_ignore_fg_state");
+    if(num == 0)
+    {
+      dt_action_t *toggle = dt_action_section(DT_ACTION(self), N_("toggle"));
```

Comment:
@AlicVB 
There's no need to put this one action into a section on its own. You can just delete the line above and directly
`dt_action_define(DT_ACTION(self), NULL, N_("sort direction"), sort->direction, &dt_action_def_toggle);`

Note the N_
Otherwise shortcuts will break when you change languages.